ota-plus-web/app/reactapp/src/components/device/HardwarePanel.jsx

```diff
@@ -45,7 +45,7 @@ class HardwarePanel extends Component {
             hardwareOverlayShown, 
             hardwareOverlayAnchor, 
             showHardwareOverlay, 
-            hideHardwareOverlay 
+            hideHardwareOverlay,
```

Comment:
Is this right?

Reply:
Actually it doesn't matter whether we'll use a comma here or not. But I'll remove it, following the standards,